src/pages/NavBar.tsx

```diff
@@ -116,7 +111,6 @@ interface pagesProps {
 
 function NavItems({ pages }: pagesProps) {
   const location = useLocation()
-  // const [anchorElNav, setAnchorElNav] = React.useState<null | HTMLElement>(null);
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const [anchorElNav, setAnchorElNav] = React.useState<null | HTMLElement>(null)
 
```

Comment:
do we need the `no-unused-vars` right below this line?

Reply:
yeah i will look into this